test/e2e/e2e.md

```diff
@@ -67,19 +67,19 @@ Meridio should test with the operator before each release to ensure the compatib
 
 9. Install the operator
 
-    ```bash
-    # update the Meridio Operator submodule
-    git pull
-    git submodule sync --recursive
-    git submodule update --init --recursive
-    # deploy certificate manager
-    kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
-    # run at the git root directory
-    cd Meridio-Operator
-    # make sure the operator image is on the cluster or can be pulled by the cluster
-    make deploy NAMESPACE=$ns
-    kubectl apply -f ./test/e2e/manifest/meridio-trench-a.yaml
-    ```
+   ```bash
+   # update the Meridio Operator submodule
+   git pull
+   git submodule sync --recursive
+   git submodule update --init --recursive
+   # deploy certificate manager
+   kubectl apply -f https://github.com/jetstack/cert-manager/releases/latest/download/cert-manager.yaml
+   # run at the git root directory
+   cd Meridio-Operator
+   # make sure the operator image is on the cluster or can be pulled by the cluster
+   make deploy NAMESPACE=$ns
```

Comment:
The image might be required, I tried it and I get: "Failed to pull image "controller:0.0.1""

Reply:
the comment in line 79 says "make sure the operator image is ..." 
maybe I should make it an explicit step